plugins/vlc-video/vlc-video-plugin.h

```diff
@@ -1,16 +1,16 @@
 #include <obs-module.h>
-#include <libvlc.h>
+#include <vlc/libvlc.h>
```

Comment:
Can you add instructions, what a developer should download and use to get VLC support compiled?

I downloaded the [7z archive](https://download.videolan.org/pub/videolan/vlc/2.2.4/win64/vlc-2.2.4-win64.7z) and used the sdk folder. But I could not find any way to get it working without changing the includes.